<?xml version="1.0"?>

<launch>


<!-- run rviz -->
<!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find core_project)/rosbags/veg_img.rviz"/> -->

<!-- Run Config File  -->
<rosparam file="$(find dalsa_genie_nano_c2420)/config/camera.yaml" />

<!-- Convert image from compressed to raw -->
<!-- <node name="dalsa_720p" type="republish" pkg="image_transport" output="screen" args="compressed in:=/dalsa_camera_720p raw out:=/compressed_720p/image_raw" /> -->
<!-- <node name="dalsa_4k" type="republish" pkg="image_transport" output="screen" args="compressed in:=/dalsa_camera raw out:=/compressed_4k/image_raw" /> -->


<!-- Run NDVI Converter node -->
<node pkg = "dalsa_genie_nano_c2420" type = "ndvi_multispectral.py" name = "ndvi_multispectral" output = "screen"/>





</launch>